contentEditable.html

```diff
@@ -52,6 +52,7 @@
       <p>
       </p>
     </section>
+    <section id='conformance'></section>
```

Comment:
@marcoscaceres Is this the right thing to do here? We implicitly mark all sections as informative without this and it causes some warnings, maybe ReSpec should give some guide for this.

Reply:
Yes to both... we need to detect if the document is on the W3C Rec track, and if it's not a note. And then including a conformance section should be sufficient. 

However, could you please move the conformance section to the end of the document? Just before any appendix.